styles/carousel.css

```diff
@@ -0,0 +1,273 @@
+.cards-container {
+    display: flex;
+    padding: 0 72px;
+    background-color: #fff;
+    box-shadow: none;
+  }
+  
+  .cards-container>.default-content-wrapper {
```

Comment:
All root selector should starts with .carousel as it is specific to carousel. I see all root selectors are .cards-container which will collide with cards component

Reply:
Changes done